drivers/timer/stm32_lptim_timer.c

```diff
@@ -86,6 +86,11 @@ static struct k_spinlock lock;
 #endif
 #endif /* !CONFIG_STM32_LPTIM_TICK_FREQ_RATIO_OVERRIDE */
 
+static inline bool ARRM_STATE_GET(void)
```

Comment:
If this is a function, please respect coding style and use lower case.
Also, name should be reviewed (would have been the case for macro as well anyway): `counter_rollover`, `autoreload_event`, ...

Reply:
I don't understand which names you want I review. Is the comment line 249 you want I improve?
```c
		/* In case of counter roll-over, add the autoreload value,
		 * because the irq has not yet been handled
		 */
```

Reply:
nevermind